great_expectations/render/view/templates/markdown_text.j2

```diff
@@ -0,0 +1,13 @@
+{% if not content_block_body_styling and content_block.get("styling") %}
+    {% set content_block_body_styling = content_block.get("styling") | render_styling %}
+{% endif %}
+
+{% include 'markdown_content_block_header.j2' %}
+{% for text in content_block["text"] -%}
+  {% if text is mapping and "styling" in text -%}
+    {% set paragraph_styling = text["styling"].get("parent", {}) | render_styling -%}
+  {% else -%}
+    {% set paragraph_styling = "" -%}
+  {% endif -%}
+  {{text | render_content_block }}
+{% endfor -%}
```

Comment:
`</div>` tags have been removed as per @anthonyburdi 's review :)

Reply:
Sorry to not be clear @Shinnnyshinshin - but I think the html styling should be removed as well. So for example, the meta notes added to expectations (which use markdown) should be rendered as-is.